es-ds-components/components/es-file-input.vue

```diff
@@ -0,0 +1,334 @@
+<template>
+    <div
+        role="presentation"
+        class="es-file-upload align-items-center rounded-sm d-flex flex-column justify-content-center p-150"
+        :class="{ 'active': active }"
+        @dragenter.stop.prevent="active = true"
+        @dragover.stop.prevent="active = true"
+        @dragleave.stop.prevent="active = false"
+        @dragend.stop.prevent="onDrop"
+        @drop.stop.prevent="onDrop"
+        @click.self="openFilePicker"
+        @keypress.enter.prevent="openFilePicker"
+        @keypress.space.prevent="openFilePicker">
```

Comment:
The same issue existed in the previous one, but can we take the opportunity to switch `keypress` to `keydown` since keypress is [deprecated and may be removed by browsers](https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event)?